app/views/components/_markdown-editor.html.erb

```diff
@@ -0,0 +1,52 @@
+<%
+  label_text ||= "Enter your markdown here"
+  textarea ||= {}
+  error_items ||= []
+   component_classes = %w[app-c-markdown-editor govuk-form-group]
+  component_classes << "govuk-form-group--error" if error_items.any?
```

Comment:
How is `error_items` being populated?

Reply:
Removed this as we're not throwing errors for this field at the moment.